timegrinder/processors.go

```diff
@@ -180,6 +180,7 @@ type Processor interface {
 	ToString(time.Time) string
 	ExtractionRegex() string
 	Name() string
+	SetCutoff(time.Time)
```

Comment:
i think we nee durations not time.Time everywhere, a floating "window around now" not some concrete timestamp.